recipes/mosquitto/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "mosquitto" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mosquitto.org/files/source/{{ name }}-{{ version }}.tar.gz
+  sha256: d0dde8fdb12caf6e2426b4f28081919a2fce3448773bdb8af0d3cd5fe5776925
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+
+  host:
+    - openssl
+    - libxslt
+
+
+test:
+  commands:
+    - mosquitto_sub --help || true  # [not win]
+
+about:
+  home: https://mosquitto.org/
+  license: OTHER
+  license_family: OTHER
```

I see you've already partly tried this. Maybe the `license_family` field being invalid caused isssues:
```suggestion
  license: EPL-2.0
```

If it still fails I'll look into the linter for why.